src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.CodeAnalysis.Remote.Workspaces</id>
```

Comment:
Wouldn't our normal naming convention be `Microsoft.CodeAnalysis.Workspaces.Remote`?